eng-wasm/src/internal_std.rs

```diff
@@ -8,6 +8,7 @@ pub use self::std::{
     str::from_utf8,
     string::{String, ToString},
     vec::Vec,
+    vec,
```

Comment:
Note that this exposes the entire `vec` namespace, not just the `vec!` macro.
If this is not ok with you, you can instead use `#[macro_use(vec)]`.
[This is explained here](https://doc.rust-lang.org/reference/macros-by-example.html#the-macro_use-attribute).

Reply:
Done (with your help :))